### PR TITLE
Updating bastion for SNI changes

### PR DESCRIPTION
**Pull request by deepsm007**
Follow up of SNI changes for sshd-bastion

Signed-off-by: Deep Mistry

### CODE REVIEW

**Comment by deepsm007** [comment]
cc @andymcc

**Comment by jaypoulz** [comment]
Very satisfying patch. :)

File: clusters/build-clusters/01_cluster/sshd-bastion/ppc64le-libvirt/deployment.yaml [hunk 1]
```diff
@@ -42,19 +42,19 @@ items:
     ports:
```
Comment (andymcc) [inline clusters/build-clusters/01_cluster/sshd-bastion/ppc64le-libvirt/deployment.yaml:42]: I think we're missing the libvirt-1 port here (for sshd-1)

File: clusters/build-clusters/01_cluster/sshd-bastion/ppc64le-libvirt/deployment.yaml [hunk 2]
```diff
@@ -65,226 +65,13 @@ items:
...
```
Comment (andymcc) [inline clusters/build-clusters/01_cluster/sshd-bastion/ppc64le-libvirt/deployment.yaml:65]: Missing libvirt-2 here

File: clusters/build-clusters/01_cluster/sshd-bastion/z/deployment.yaml [hunk 3]
```diff
@@ -42,200 +42,13 @@ items:
     ports:
     - name: api
```
Comment (andymcc) [inline clusters/build-clusters/01_cluster/sshd-bastion/z/deployment.yaml:43]: libvirt-1 missing here

**Review by andymcc — CHANGES_REQUESTED** [review]
I think it Looks good, but we're missing the libvirt ports

**Pull request merged**